### PR TITLE
Emotes

### PR DESCRIPTION
**Pull request by ultraviolet-jordan**
Animation

### CODE REVIEW

File: data/src/scripts/controls/player_controls.rs2 [hunk 1]
```diff
@@ -3,3 +3,46 @@
...
+
+[if_button,player_controls:com_24]
+~do_emote(emote_clap);
+
+[proc,do_emote](seq $anim)
+anim($anim, 0);
+// TODO: "You're already doing an emote!"
+def_int $client_frames = calc(seqlength($anim) * 20);
+def_int $delay = calc($client_frames / 600);
+p_delay($delay);
```
Comment (Jameskmonger) [inline data/src/scripts/controls/player_controls.rs2:48]: not too sure if it matters for `.rs2` but this file doesn't end in a newline

**Review by Jameskmonger — APPROVED** [review]
LGTM overall, one comment. Nice work

File: data/src/scripts/controls/player_controls.rs2 (hunk 1 above)
Comment (Pazaz) [reply data/src/scripts/controls/player_controls.rs2:48]: doesn't break the compiler, but would be good for consistency

Comment (ultraviolet-jordan) [reply data/src/scripts/controls/player_controls.rs2]: Fixed.

**Pull request merged**